dataset/dictionary/characters.json5

```diff
@@ -1173,6 +1176,7 @@
     en: "Khedive Pursina",
     ja: "ヘディーヴ・プルシナ",
     // zhCN: "海迪夫", // TODO zh-translation for Pursina
+    // PS: It seems that official translation of Pursina is not provided in the game.
```

Comment:
Maybe this?
![Khedive Pursina's Chinese translation](https://user-images.githubusercontent.com/107226297/187026001-4a1134a3-d5a6-41c9-abdd-4136fddb0bd1.png)